src/main/java/pl/med/clinic/service/PatientServiceImpl.java

```diff
@@ -52,4 +53,8 @@ public PatientDtoResponse updatePatient(Long id, PatientDtoRequest updatedPatien
     public void deletePatientById(Long id) {
         patientRepository.deleteById(id);
     }
+
+    public LocalDate getAge(Long id){
+        return patientRepository.getOrThrow(id).getDateOfBirth();
+    }
```

Comment:
ta metoda nigdzie nie jest używana, a jeżeli wywołujesz ją w tej klasie to zawsze powinna być private, nigdy metody publiczne nie powinny wywoływać metod publicznych z tych samych klas (CLEAN CODE)

Reply:
usunęłam metodę, chciałam sprawdzać jak działa, ale w danych responsie wyglądało dobrze